src/SprykerEco/Shared/Billpay/BillpayConstants.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Shared\Billpay;
+
+interface BillpayConstants
+{
+    const GATEWAY_URL = 'BILLPAY:GATEWAY_URL';
+    const IS_TEST_MODE = 'BILLPAY:IS_TEST_MODE';
+    const VENDOR_ROOT = 'BILLPAY:VENDOR_ROOT';
+
+    const BILLPAY_MERCHANT_ID = 'BILLPAY:BILLPAY_MERCHANT_ID';
+    const BILLPAY_PORTAL_ID = 'BILLPAY:BILLPAY_PORTAL_ID';
+    const BILLPAY_SECURITY_KEY = 'BILLPAY:BILLPAY_SECURITY_KEY';
+    const BILLPAY_PUBLIC_API_KEY = 'BILLPAY:BILLPAY_PUBLIC_API_KEY';
+    const BILLPAY_MAX_DELAY_IN_DAYS = 'BILLPAY:BILLPAY_MAX_DELAY_IN_DAYS';
+    const USE_MD5_HASH = 'BILLPAY:USE_MD5_HASH';
+
+    const USE_PRESCORE = 'BILLPAY:USE_PRESCORE';
+
+    const PARAM_GROUP_CUSTOMER = 'BILLPAY:PARAM_GROUP_CUSTOMER';
```

Comment:
Why are all those env configs? They shouldnt, as they dont change per env, do they?
Only test mode/gateway and the id/keys should be part of env config. All other should be hardcoded in real config.